### ISSUE TITLE
Binary testing: "No configuration files"

### ISSUE THREAD

**Issue by kmoe** [hashicorp/terraform-plugin-sdk#495]
Version: 2.0.0-rc.2

Example log on windows:

```
=== RUN   TestAccResourceID
cannot run Terraform provider tests: failed to create symlink at C:\Users\paul\AppData\Local\Temp\tftest-terraform-provider-random607016463\plugins-current035595298\terraform-provider-random to C:\Users\paul\AppData\Local\Temp\go-build128261702\b233\provider.test.exe: symlink C:\Users\paul\AppData\Local\Temp\go-build128261702\b233\provider.test.exe C:\Users\paul\AppData\Local\Temp\tftest-terraform-provider-random607016463\plugins-current035595298\terraform-provider-random: A required privilege is not held by the client.
FAIL    github.com/terraform-providers/terraform-provider-random/internal/provider      0.321s
```

**Comment by paultyng** [comment]
This works though when in an escalated shell, so that is the current workaround.

**Comment by paddycarver** [comment]
With an escalated shell I still get an error about no configuration, let me reboot and I'll capture it here.

**Comment by paddycarver** [comment]
Running as Administrator, in Developer Mode:

```
--- FAIL: TestAccResourceID (1.89s)
    testing_new.go:94: terraform failed: exit status 1
        
        stderr:
        
        Error: No configuration files
        
        Apply requires configuration to be present. Applying without a configuration
        would mark everything for destruction, which is normally not what is desired.
        If you would like to destroy everything, run 'terraform destroy' instead.
```

**Comment by paultyng** [comment]
Hmm, it works on my machine with just a "Run as administrator" Powershell instance. Don't think I'm using the developer mode though maybe.

**Comment by paddycarver** [comment]
OK, so after some fun debugging, we figured this one out. Probably. It _looks like_ the root cause is hashicorp/terraform#25233 doesn't apply _just_ to plan files, but _also_ to the config directories if you pass them in manually. Which... we are.

We confirmed this works in 0.13.0-beta2 and 0.13.0-beta3, which makes sense, as hashicorp/terraform#25233 was fixed in 0.13.0-beta2.

I've _also_ confirmed that this works in 0.12.26 and higher if we use a relative path when specifying the config directory, instead of an absolute path. Given that hashicorp/terraform#25233 is about specifying things by relative path, this seems to further point to it as the root of the issue.

Given that we can fix this SDK-side without _too much_ effort, I'm proposing we move this back into the 2.0.0 release and instead modify the SDK to use relative paths for the config files. I'll have a PR up for that shortly.
